lib/widget/training_information/post-op-home/daily_activity_advice.dart

```diff
@@ -36,6 +37,21 @@ class DailyActivityAdvice extends StatelessWidget {
                             padding: const EdgeInsets.only(top: 8),
                             child: Text(
                                 '''การปฏิบัติกิจวัตรประจำวันภายหลังการผ่าตัดช่องท้อง สามารถทำได้ดังนี้'''),
+                          ),
+                           Container(
+                            margin: EdgeInsets.symmetric(horizontal: 4),
+                            child: YoutubePlayer(
+                              controller: YoutubePlayerController(
+                                initialVideoId: YoutubePlayer.convertUrlToId(
+                                    "https://youtu.be/APUgLilx37A"),
```

Comment:
Is there any other way to store youtube link? like in firebase storage

Reply:
^^^ is optional

Reply:
Or try to store all youtube links into one file. and named it correctly so that we don't have to open the link ourselves to identify which is which.